pkg/plugins/ifaces.go

```diff
@@ -164,6 +164,7 @@ func (fn ClientMiddlewareFunc) CreateClientMiddleware(next Client) Client {
 
 type FeatureToggles interface {
 	IsEnabled(flag string) bool
+	GetEnabled(ctx context.Context) map[string]bool
```

Comment:
is this required?  if yes, could/should it be an array?  (do we send false values?)

Reply:
Since this is all internal, it maybe does not matter

Reply:
In this interface I'm only exposing a method that already exists in `FeatureManger`
https://github.com/grafana/grafana/blob/main/pkg/services/featuremgmt/manager.go#L132

If I want to return an array I would have to include a new method in `FeatureManager`